spec/board_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require_relative './../lib/board.rb'
+
+RSpec.describe Board do
+  describe '#available?' do
+    it 'return true if the move is available' do
+      board = Board.new
+      expect(board.available?(4)).to eql(true)
+    end
+  end
+end
```

Comment:
You should also test for a `false` case so you make sure you are not having a false positive.